centreon/www/include/configuration/configObject/command/DB-Func.php

```diff
@@ -203,12 +203,14 @@ function updateCommand($cmd_id = null, $params = array())
             ? $ret["command_activate"]["command_activate"]
             : null;
 
+    $type = $isCloudPlatform ? $ret['type'] : $ret["command_type"]["command_type"];
```

Comment:
When using an associative array, it's recommended to do a isset() of all the keys before using it because a value can be null. Are you sure to have no null ? I usually do a isset() of all the keys to be sur to have no errors.

Reply:
It's ok for me